penguin/llm/adapters/anthropic.py

```
@@ -6,11 +6,10 @@
 import os
 import time
 import traceback
-from typing import Any, Dict, List, Optional, Tuple, AsyncIterator, Callable, Union
+from typing import Any, Callable, Dict, List, Optional, Set, Tuple, Union
```

Ruff (0.15.12)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)